storage/paths/local.go

```diff
@@ -238,7 +241,7 @@ func (st *Local) OpenPath(ctx context.Context, p string) error {
 	if err != nil {
 		return err
 	}
-
+	ctx, _ = tag.New(ctx, tag.Upsert(metrics.Network, buildconstants.NetworkBundle))
```

Comment:
the two in this file feel far to deep to be adding this global tag, what happens if you remove these? Do they show up in your `curl`?